Dockerfile

```diff
@@ -9,12 +9,24 @@ RUN \
     lsblk \
     dmidecode \
     util-linux \
-    lm-sensors \
-    speedtest-cli &&\
+    lm-sensors &&\
   apk --no-cache add \
     raspberrypi \
     || true
```

Comment:
This part should also only run on ARM - I ignored the errors on amd64, but I think it would be best to include it below.

Reply:
Sure, that will be better.